ecs/states-inventory-hardware/docs/README.md

```diff
@@ -1,3 +1,17 @@
+## `wazuh-states-inventory-hardware` index data model
+
+### Fields summary
+
+The fields are based on https://github.com/wazuh/wazuh/issues/27898
+
+Based on ECS:
+
+- [File Fields](https://www.elastic.co/guide/en/ecs/current/ecs-file.html).
```

Comment:
These fields are based on ECS Host fields, not ECS File fields
```suggestion
- [Host Fields](https://www.elastic.co/guide/en/ecs/current/ecs-host.html).
```

Reply:
Fixed on commit https://github.com/wazuh/wazuh-indexer-plugins/pull/473/commits/98e6f7d7fb9f80ad6d243a15b9080dfb57cf522f